### PR TITLE
CLOUDP-59567: Adding the support of alerts for OM in mcli

### PR DESCRIPTION
**Pull request by andreaangiolillo**
_Jira ticket:_ [CLOUDP-59567](https://jira.mongodb.org/browse/CLOUDP-59567)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [x] **I have tested it using my local ops manager**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,8 +11,9 @@ require (
 	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/mongodb/go-client-mongodb-atlas v0.1.4-0.20200318163834-8b6a5ea6eb67
-	github.com/mongodb/go-client-mongodb-ops-manager v0.0.2-0.20200319100114-648fc659365b
+	github.com/mongodb/go-client-mongodb-atlas v0.1.4-0.20200323160302-12cb92c69adf
+	github.com/mongodb/go-client-mongodb-ops-manager v0.0.2-0.20200324155204-287d08b2e57b
+	github.com/mwielbut/pointy v1.1.0
```
Comment (gssbzn) [inline go.mod]: any reason to include pointy now? can you please check this? will it compile if we remove this line?

Comment (gssbzn) [reply go.mod]: I think I'm getting this on master as well, so may as well 🤷‍♂

Comment (andreaangiolillo) [reply go.mod]: `go mod tidy` does not clean pointy but the application compiles without it. I do not how ponty went there 🤦‍♂

Comment (andreaangiolillo) [reply go.mod]: If I remove ponty from go.mod and then run `make fmt check build`, ponty is again inside go.mod

Comment (gssbzn) [reply go.mod]: Found the way, you added it to a fixture

**Pull request merged**